README.md

```diff
@@ -66,6 +66,50 @@ Engagement.enablePushNotifications()
 
 Note: Android doesn't require any permissions in order to enable push notifications. If you feel you don't need to display any information about this to your user, you can simply call the above method immediately after calling `init()`.
 
+##### Opening your app from push notifications
+To open your app after tapping a push notification you need to add the following Intent filter to your Launcher Activity (usually your Main Activity):
+```xml
+<intent-filter tools:ignore="AppLinkUrlError">
+    <category android:name="android.intent.category.DEFAULT" />
+    <action android:name="android.intent.action.VIEW" />
+    <data android:mimeType="engagement/message" />
+</intent-filter>
+```
+
+Note: The chosen Activity should be using `android:launchMode="singleTop"`.
+
+If the Activity is already running, the `onNewIntent(Intent intent)` method will be called and you should handle the Intent there:
+```kotlin
+override fun onNewIntent(intent: Intent?) {
+    super.onNewIntent(intent)
+    setIntent(intent)
+    // Handle the intent here.
+}
+```
+
+If the Activity isn't running, a new instance will be created and you should handle the Intent in `override fun onCreate(savedInstanceState: Bundle?)`:
```

Comment:
do we need the `override fun` part here? It doesnt match the function description we have for `onNewIntent`

Reply:
Updated. Thanks brother!